capabilities/grafana/zarf.yaml

```
@@ -15,13 +15,14 @@ components:
         namespace: grafana
         valuesFiles:
           - values/values.yaml
-    images:
-      - docker.io/grafana/grafana:10.1.5
-
-  - name: uds-grafana-config
-    required: true
-    charts:
       - name: uds-grafana-config
         namespace: grafana
         version: 0.1.0
         localPath: chart
+    images:
+      - docker.io/grafana/grafana:10.1.5
+      - docker.io/curlimages/curl:7.85.0
+      - docker.io/grafana/grafana-image-renderer:latest
+      - docker.io/library/busybox:1.31.1
+      - docker.io/bats/bats:v1.4.1
```

```suggestion
      - docker.io/library/busybox:1.31.1
```
I don't think we use either of these two - image renderer is disabled and the bats image is for their test framework.